modules/ask_va_api/app/lib/ask_va_api/inquiries/creator.rb

```diff
@@ -75,6 +75,27 @@ def log_safe_fields_from_inquiry(inquiry_params)
         inquiry_params.slice(*SAFE_INQUIRY_FIELDS)
```

Comment:
Is there a high cardinality risk for the safe fields? Even with "safe" fields, if the values of those fields are very diverse, it could be expensive. Unique fields=higher cost

Reply:
You are absolutely right. Didn't think about that. Would withdraw this PR. Thanks for pointing it out. I need to log some very targeted fields.